homeassistant/components/spokestack_wakeword/manifest.json

```diff
@@ -0,0 +1,15 @@
+{
+  "domain": "spokestack_wakeword",
+  "name": "Spokestack Wakeword",
+  "documentation": "https://www.home-assistant.io/integrations/spokestack_wakeword",
+  "config_flow": true,
+  "requirements": [
+    "tensorflow==2.3.0",
```

Comment:
We no longer run Tensorflow inside the Home Assistant container. The compilation with Alpine was a nightmare to maintain.

I also think in the bigger scope that wakeword detection shouldn't be part of Home Assistant. 

People have their HA box in a closet. The use case for this is very small. It should instead be focused on standalone devices people will have in their living rooms, like what we made available with [Ada](https://github.com/home-assistant/ada). Although Ada never took off and Almond, for which we build it, is now doing STT and TTS themselves.

Reply:
We have some users that would like to use our wakeword service on home assistant so we would like to build something they could use.

Would the correct approach be an add-on instead of an integration? As long as the add-on can trigger the event like I've done here?

If I'm not mistaken that would fix the TensorFlow issue as well.

Thank you so much for taking the time to look at this, I really appreciate it!

Reply:
Yes, an add-on is fine. And you can just fire events through the rest API. The add-on will have access to Home Assistant API via a token set as environment variable.